src/Template.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace Jh\Workflow;
+
+/**
+ * @author Michael Woodward <michael@wearejh.com>
+ * @author Aneurin "Anny" Barker Snook <anny@wearejh.com>
+ */
+class Template
+{
+    const TEMPLATE_DIR = __DIR__ . '/../templates/';
+
+    public function cp($template, $dest)
+    {
+        $this->provision(dirname($dest));
+        copy(static::TEMPLATE_DIR . $template, $dest);
+    }
+
+    private function provision($dest, $of = '/')
+    {
+        $path = '';
+        $d = explode('/', ltrim($dest, '/'));
+        foreach ($d as $dd) {
```

Comment:
isn't what this is doing just an option on `mkdir` to do it recursivly ?

Reply:
I can't believe I've been a PHP developer for ten years now and never realised that `mkdir` has a `$recursive` argument